README.md

```diff
@@ -30,6 +30,217 @@ you're a tool author you may be interested in the formal [EBNF grammar][].
 [EBNF grammar]: https://github.com/projectfluent/fluent/tree/master/spec
 
 
+Installation
+------------
+
+    pip install fluent
+
+Usage
+-----
+
+Fluent uses unicode everywhere internally, and doesn't accept bytestrings, so
+that you avoid problems down the road when non-ASCII characters start appearing
+in FTL files. If you are using Python 2, for the following examples you will
+first need to do:
+
+    >>> from __future__ import unicode_literals
+
+or add unicode literal markers to strings.
+
+To generate translations from this Python libary, you start with the
+`MessageContext` class:
```

Comment:
It would be cool to update the naming scheme to the current one, but I'm ok doing that in a follow up.

Reply:
I've now updated it `FluentBundle`, but as part of the namespace package work - see #81 . I did that as a separate PR because all the renames will no doubt confuse this code review terribly!